lib/open_telemetry/tracer.ex

```diff
@@ -0,0 +1,45 @@
+defmodule OpenTelemetry.Tracer do
+  @moduledoc """
+
+      use OpenTelemetry.Tracer
```

Comment:
You don't need this line?

Reply:
Ah right, now that we don't use `using` it can just be `require`?

Reply:
Yes!

Reply:
Cool, thanks, fixed. That's what I get for just copying around parts from the version that created a new tracer module, so needed `use` and to use that module's name, not the one using the macros :)